lib/active_support/cache/memcached_store.rb

```diff
@@ -247,13 +247,21 @@ def handle_exceptions(return_value_on_error:, on_miss: return_value_on_error)
       rescue Memcached::NotFound, Memcached::ConnectionDataExists
         on_miss
       rescue Memcached::Error => e
-        logger.warn(
-          "[MEMCACHED_ERROR] swallowed=#{@swallow_exceptions} exception_class=#{e.class} exception_message=#{e.message}"
-        ) if logger
+        log_warning(e)
         raise unless @swallow_exceptions
         return_value_on_error
       end
 
+      def log_warning(err)
+        return unless logger
+        return if err.is_a?(Memcached::NotStored) && @swallow_exceptions
+
+        logger.warn(
```

Comment:
Is not better to always log this as debug level? I wonder if there is any case where we are shallowing exceptions but we still want to know what is the exception being shallowed.

Reply:
I think now it makes more sense to keep it as a warning as most of the errors will be ring error/timeouts/connection errors now.

Reply:
I assume you want to do this because `NotStored` is *very* common and "fine", and you're still interested in other, less common errors. What's the next most common error logged from here?

Reply:
Mostly `ServerIsMarkedDead` and `ATimeoutOccurred`. Currently we set swallow=true, yet we use `unless_exists: true` in many places, which causes this error to generate a lot of noise.